dali/test/python/test_dali_tf_dataset_eager.py

```diff
@@ -316,22 +344,22 @@ def check_layout(kwargs, input_datasets, layout):
 
     with tf.device('/cpu:0'):
         dali_dataset = dali_tf.experimental.DALIDatasetWithInputs(
-                input_datasets=input_datasets,
-                pipeline=pipe,
-                batch_size=pipe.max_batch_size,
-                output_shapes=None,
-                output_dtypes=tf.int64,
-                num_threads=pipe.num_threads,
-                device_id=pipe.device_id)
+            input_datasets=input_datasets,
+            pipeline=pipe,
+            batch_size=pipe.max_batch_size,
+            output_shapes=None,
+            output_dtypes=tf.int64,
+            num_threads=pipe.num_threads,
+            device_id=pipe.device_id)
 
     run_dataset_eager_mode(dali_dataset, 10)
 
 
 def run_tf_with_dali_external_source(dev, es_args, ed_dev, dtype, *_):
     run_tf_dataset_eager_mode(dev,
-        get_pipeline_desc=get_external_source_pipe(es_args, dtype, ed_dev),
-        to_dataset=external_source_to_tf_dataset,
-        to_stop_iter=True)
+                              get_pipeline_desc=get_external_source_pipe(es_args, dtype, ed_dev),
+                              to_dataset=external_source_to_tf_dataset,
+                              to_stop_iter=True)
```

Comment:
The previous indentation in these lines was nice, we can keep it. We can keep the new one as well. Again, your call.